supervisor_haproxy/haproxy_control.py

```diff
@@ -66,7 +67,14 @@ def command(self, cmd):
     @contextmanager
     def connect(self):
         sock = socket.socket(self.sock_family, socket.SOCK_STREAM)
-        sock.connect(self.sock_address)
+        try:
+            sock.connect(self.sock_address)
+        except socket.error, exc:
+            if exc.args[1] == 'Connection refused':
```

Comment:
Instances of `socket.error` should have an attribute that can be compared against [`errno.ECONNREFUSED`](
https://docs.python.org/2/library/errno.html#errno.ECONNREFUSED) I think, instead of string comparison.